corehq/apps/couch_sql_migration/couchsqlmigration.py

```diff
@@ -492,6 +496,10 @@ def _filter_case_diffs(doc_type, diffs):
         if diff.path not in const.CASE_IGNORED_PATHS and diff not in const.CASE_IGNORED_DIFFS
     ]
     filtered_diffs = _check_deletion_fields_date(filtered_diffs, doc_type)
+    filtered_diffs = [
+        diff for diff in filtered_diffs
+        if diff.path[0] == 'owner_id' and diff.old_value is None
+    ]
```

Comment:
was the intention that only the `owner_id` ends up as a possible diff? why would that be and what's the point of the other checks?